### PR TITLE
Update Proto Source files with latest proto definitions

### PR DESCRIPTION
**Pull request by anshuldata**
* Fixed name of Join type (it was a breaking change in substrait)

### CODE REVIEW

File: src/from_substrait.cpp [hunk 1]
```diff
@@ -430,10 +430,10 @@ shared_ptr<Relation> SubstraitToDuckDB::TransformJoinOp(const substrait::Rel &so
 	case substrait::JoinRel::JoinType::JoinRel_JoinType_JOIN_TYPE_RIGHT:
 		djointype = JoinType::RIGHT;
 		break;
-	case substrait::JoinRel::JoinType::JoinRel_JoinType_JOIN_TYPE_SINGLE:
+	case substrait::JoinRel::JoinType::JoinRel_JoinType_JOIN_TYPE_LEFT_SINGLE:
 		djointype = JoinType::SINGLE;
 		break;
-	case substrait::JoinRel::JoinType::JoinRel_JoinType_JOIN_TYPE_SEMI:
+	case substrait::JoinRel::JoinType::JoinRel_JoinType_JOIN_TYPE_LEFT_SEMI:
```
Comment (EpsilonPrime) [inline src/from_substrait.cpp:436]: Should we add the other new join types to this switch statement?  Even though we aren't handling it yet it's probably worth adding it explicitly to the default clause.

**Review by EpsilonPrime — APPROVED** [review]

File: src/from_substrait.cpp (hunk 1 above)
Comment (anshuldata) [reply src/from_substrait.cpp:436]: Added missing join types explicitly

**Comment by anshuldata** [comment]
missed adding missing join to file to_substrait. Update now. Can you re-approve @EpsilonPrime

**Comment by anshuldata** [comment]
@EpsilonPrime I don't have write access to this branch. please merge this.
Thanks

**Pull request merged**